### PR TITLE
Use dep for vendoring, add opentracing instrumentation.

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: scripts/vendor_health.sh [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
+set -eu
 
 if ! grep -q $GOPATH/bin <<< $PATH ; then
   export PATH="$PATH:$GOPATH/bin"
 fi
 
-if ! which govendor >/dev/null; then
-	go get github.com/kardianos/govendor || exit 1
+if ! which dep >/dev/null; then
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```
Comment (Dieterbe) [inline scripts/vendor_health.sh]: MT runs `go get -u github.com/golang/dep/cmd/dep` which works fine.
https://github.com/grafana/metrictank/blob/master/scripts/qa/vendor.sh

**Comment by tomwilkie** [comment]
This doesn't work, and the macaron router is so convoluted that I can't figure out why.

File: scripts/vendor_health.sh (hunk 1 above)
Comment (tomwilkie) [reply scripts/vendor_health.sh]: From https://github.com/golang/dep:
> It is strongly recommended that you use a released version.

**Comment by Dieterbe** [comment]
I think it's important for a PR that changes vendored dependencies to be more clear about those changes. I created a new PR, see above link. I also didn't have any issues with macaron.

**Pull request closed without merging**